src/core/server/locales/en-US/email.ftl

```diff
@@ -24,6 +24,13 @@ email-notification-template-passwordChange =
 
 email-subject-passwordChange = Your password has been changed
 
+email-subject-updateUsername = Your username has been changed
+
+email-notification-template-updateUsername =
+  Hello { $username },<br/><br/>
+  Thank you for updating your {{ context.organizationName }} commenter account information. The changes you made are effective immediately. <br /><br />
```

Comment:
Looks like `{{ context.organizationName }}` was copied from the HTML template! It should be `{ $organizationName }`